apps/recovery-portal/src/main/webapp/self-registration-complete.jsp

```diff
@@ -85,6 +135,28 @@
             <%=IdentityManagementEndpointUtil.i18n(recoveryResourceBundle, "Close")%>
         </button>
     </div>
+    <form id="callbackForm" name="callbackForm" method="post" action="/commonauth">
+        <%
+            if (username != null) {
+        %>
+        <div>
+            <input type="hidden" name="username"
+                   value="<%=Encode.forHtmlAttribute(username)%>"/>
+        </div>
+        <%
+            }
+        %>
+        <%
+            if (sessionDataKey != null) {
+        %>
+        <div>
+            <input type="hidden" name="sessionDataKey"
+                   value="<%=Encode.forHtmlAttribute(sessionDataKey)%>"/>
```

Comment:
Can't have this in the previous line itself?

Reply:
fixed [ed5eadb](https://github.com/wso2/identity-apps/pull/1409/commits/ed5eadbb4f3cd1e376311a9ace165eef81e761c6)